test/basic.html

```diff
@@ -165,6 +165,28 @@
           }, 200);
         });
 
+        test('clicking the selected item should deselect if allow-empty-selection is set', function (done) {
+          var g = fixture('WithSelection');
+          g.allowEmptySelection = true;
+          var items = g.items;
+
+          expect(items[0].checked).to.be.equal(true);
+          MockInteractions.tap(items[0]);
+
+          // The selection should not change, but wait for a little bit just
+          // in case it would and an event would be fired.
+          setTimeout(function() {
+            try {
+              expect(items[0].checked).to.be.equal(false);
+              expect(items[1].checked).to.be.equal(false);
+              expect(items[2].checked).to.be.equal(false);
+              done();
+            } catch (e) {
+              done(e)
+            }
+          }, 200);
```

Comment:
I copy pasted from the other ones! Which I think were copy pasted from somewhere else. :)